addon/components/paper-switch.js

```diff
@@ -47,21 +70,6 @@ export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
     let switchHammer = new Hammer(element);
     this.switchHammer = switchHammer;
```

Comment:
Also, let's destructure `run` at the top, so that we use `run.bind` instead of `Ember.run.bind`.